app/routes/_gcn.circulars/circulars.server.ts

```diff
@@ -173,17 +201,24 @@ export async function search({
 }
 
 /** Get a circular by ID. */
-export async function get(circularId: number): Promise<Circular> {
+export async function get(
```

Comment:
You can make this function a lot simpler now:

```
/** Get a circular by ID. */
export async function get(circularId: number, version?: number) {
  const circularVersions = await getDynamoDBVersionAutoIncrement(circularId)
  const result: Circular = await circularVersions.get(version)
  if (!result)
    throw new Response(null, {
      status: 404,
    })
  return result
}
```

Reply:
Sweet, updated in the latest commit with a rebase